src/amuse/community/higpus/interface.cc

```diff
@@ -193,10 +193,10 @@ int evolve_model(double t){
    FMAX = 1000000 + (int) ((GTIME+GTW) / DTPRINT);
 	
    TTIME=t;
-   HostSafeCall(Hermite6th(TTIME, &GTIME, &ATIME, local_time, step, N, M, pos_PH, vel_PH, pos_CH, vel_CH, a_H0, MAXDIM, NGPU, TPB, rank, size, BFMAX, ETA6, ETA4, DTMAX, DTMIN, EPS, DTPRINT, FMAX, warm_start, GTW, GPUMINTHREADS, plummer_core, plummer_mass, rscale, mscale, dev, &cleanstop, path));
+   HostSafeCall(Hermite6th(TTIME, &GTIME, &ATIME, local_time, step, N, M, pos_PH, vel_PH, pos_CH, vel_CH, a_H0, MAXDIM, NGPU, TPB, rank_higpus, size_higpus, BFMAX, ETA6, ETA4, DTMAX, DTMIN, EPS, DTPRINT, FMAX, warm_start, GTW, GPUMINTHREADS, plummer_core, plummer_mass, rscale, mscale, dev, &cleanstop, path));
 
    MPISafeCall(MPI_Barrier(MPI_COMM_WORLD));
-   MPISafeCall(MPI_Bcast(&cleanstop, 1, MPI::BOOL, 0, MPI_COMM_WORLD));
+   MPISafeCall(MPI_Bcast(&cleanstop, 1, MPI_C_BOOL, 0, MPI_COMM_WORLD));
```

Comment:
Shouldnt this (and the other MPI::BOOLs) be a MPI_CXX_BOOL ? It may not matter in practice, but that depends on compiler implementation of c and c++ bools

Reply:
I think the MPI_CXX_BOOL type has been introduced very recently, and it depends on MPI 3.0. That's why I preferred to not use it. But yea, as you said, it may not matter in practice, especially for MPI_Bcast functions.

Reply:
ok, didn't realize it is that recent...

Reply:
Actually, I think they introduced these CXX types in MPI-2.2... but then MPI-2.2 was also the last version to support CXX bindings. But then, MPI-3.0 re-introduced full support for some CXX datatypes...so, better to stick with C bindings, I guess :-D